### DISCUSSION TITLE
Bulk/fast update fields in overview

### DISCUSSION THREAD

**Discussion by dakkafex** [directus/directus#8445] [Feature Requests]
I think it would be very useful if users could update (selected) fields in the overview of a collection.

Sure the interface is pretty fast and autosaving, but with our clients we keep getting requests for it if they could just quickly edit things in the oversight, i.e. the price of a product

**Comment by azrikahar** [comment]
Perhaps #3874 fit what you meant?

**Reply by benhaynes** [reply]
A full spreadsheet layout would certainly work, we could also look into right clicking a field to show a context menu with an option to edit. Clicking that could give a popover / drop-down of the interface to edit it on the table layout.

**Reply by dakkafex** [reply]
pretty much i guess yes, voted on it.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
